edx_django_utils/monitoring/README.rst

```diff
@@ -7,6 +7,13 @@ See ``__init__.py`` for a list of everything included in the public API.
 
 If, for some reason, you need low level access to the newrelic agent, please extend this library to implement the feature that you want. Applications should never include ``import newrelic.agent`` directly.
 
+Choice of monitoring tools
```

Comment:
Nit: maybe mention here that NewRelic is the default

Reply:
Maybe also mention that not only is it a list of implementations, but also that multiple implementations can work at once. That might be obvious once you look at the code, but would be helpful to those who do not have time to do so.